specification/hybridaks/resource-manager/readme.md

```diff
@@ -27,14 +27,23 @@ These are the global settings for the hybridaks.
 ``` yaml
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2024-01
+tag: package-preview-2025-02
 ```
```

Comment:
the first commit needs to be an exact copy of the previous api version. All new changes should only be added in the subsequent commits.
This is explained in detail here -  [https://eng.ms/docs/products/azure-developer-experience/design/api-specs/api-specs?tabs=openapi](https://eng.ms/docs/products/azure-developer-experience/design/api-specs/api-specs?tabs=openapi)

Reply:
It's in this commit cfe26c884459b40af138e0d7c409eb39dc8fff7d. I follow .\eng\scripts\Copy-ApiVersion.ps1 <ServiceDirectory> <ServiceType> <Provider> <NewVersion> <BaseVersion>